packages/twenty-front/src/pages/onboarding/ChooseYourPlan.tsx

```diff
@@ -156,7 +156,7 @@ export const ChooseYourPlan = () => {
     handleCheckoutSession();
```

Comment:
I think this code is dangerous. Side-effects should not be called during render. It's not in the scope of this PR but I think it should be refactored.